lighthouse-core/report/html/renderer/element-screenshot-renderer.js

```diff
@@ -139,24 +139,30 @@ class ElementScreenshotRenderer {
    * @param {LH.Artifacts.FullPageScreenshot} fullPageScreenshot
    */
   static installOverlayFeature(dom, templateContext, fullPageScreenshot) {
-    const reportEl = dom.find('.lh-report', dom.document());
-    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
-    if (reportEl.classList.contains(screenshotOverlayClass)) return;
-    reportEl.classList.add(screenshotOverlayClass);
+    const topbarEl = dom.find('.lh-topbar', dom.document());
+    const containerEl = topbarEl.parentElement;
+    if (!containerEl) throw new Error('could not find parent element');
 
-    const maxLightboxSize = {
-      width: dom.document().documentElement.clientWidth,
-      height: dom.document().documentElement.clientHeight * 0.75,
-    };
+    const screenshotOverlayClass = 'lh-feature-screenshot-overlay';
+    // Don't install the feature more than once.
+    if (containerEl.classList.contains(screenshotOverlayClass)) return;
+    containerEl.classList.add(screenshotOverlayClass);
 
-    dom.document().addEventListener('click', e => {
+    containerEl.addEventListener('click', e => {
```

Comment:
```suggestion
    // Add a single listener to the top container to handle a click on any node.
    containerEl.addEventListener('click', e => {
```

some optional explicitness :)